warehouse/models/intermediate/gtfs/gtfs_quality/int_gtfs_quality__passes_fares_validator.sql

```diff
@@ -0,0 +1,63 @@
+WITH feed_guideline_index AS (
+    SELECT * FROM {{ ref('int_gtfs_quality__schedule_feed_guideline_index') }}
+),
+
+keyed_parse_outcomes AS (
+    SELECT * FROM {{ ref('int_gtfs_schedule__keyed_parse_outcomes')}}
+),
+
+daily_feed_fare_files AS (
+    SELECT feed_key,
+           COUNT(*) AS ct_files
+      FROM keyed_parse_outcomes
+     WHERE parse_success
+       AND gtfs_filename IN ('fare_leg_rules',
+                             'rider_categories',
+                             'fare_containers',
+                             'fare_products',
+                             'fare_transfer_rules'
+                             )
+       AND feed_key IS NOT null
+     GROUP BY 1
+),
+
+validation_fact_daily_feed_codes_fares_related AS (
+    SELECT * FROM {{ ref('fct_daily_schedule_feed_validation_notices') }}
```

Comment:
Can we move this to the top as a dependency declaration?

Reply:
will do!